.test-infra/jenkins/job_PerformanceTests_FileBasedIO_IT.groovy

```diff
@@ -16,155 +16,136 @@
  * limitations under the License.
  */
 
-import CommonJobProperties as commonJobProperties
+import CommonJobProperties as common
 
-def testsConfigurations = [
+def jobs = [
         [
-                jobName           : 'beam_PerformanceTests_TextIOIT',
-                jobDescription    : 'Runs PerfKit tests for TextIOIT',
-                itClass           : 'org.apache.beam.sdk.io.text.TextIOIT',
-                bqTable           : 'beam_performance.textioit_pkb_results',
-                prCommitStatusName: 'Java TextIO Performance Test',
-                prTriggerPhase    : 'Run Java TextIO Performance Test',
-                extraPipelineArgs: [
+                name               : 'beam_PerformanceTests_TextIOIT',
+                description        : 'Runs PerfKit tests for TextIOIT',
```

Comment:
Here and all below - don't we need to change `PerfKit` name to something else? Maybe just `performance`?

Reply:
I agree - I missed this. Thanks!